actions/install-testmo/action.yml

```diff
@@ -0,0 +1,12 @@
+name: Install Testmo CLI tool
+description: Install Testmo including requirements (Node.js)
+
+runs:
+  using: composite
+  steps:
+    - uses: actions/setup-node@v4
```

Comment:
Nice to see this collapse a bunch of lines in the old action.  Are there any node version requirements/restrictions for testmo that we should worry about here?  it looks like the old code set a specific version.

Reply:
Good question. This is set to always use the latest LTS (long-term support) release.

For some information about Node.js releases, the even-numbered major version releases _eventually become_ LTS releases (after 6 months) at which point the previous LTS swaps from active to maintenance mode for a period of time. Odd-numbered major version releases’ (such as 19) support is discontinued after the 6-month period.

Given that, I have two notes:
* If Testmo ever stops supporting the current LTS release, there is something seriously wrong with their service, and we should have jumped ship a while ago.
* Pinning to a specific version could work, but would require more maintenance upkeep on our end (every year or so). Even if we preferred this path, though, an odd-numbered release should never be used, and an even number should not be used until it reaches LTS status.

Reply:
I hadn't caught the latest LTS part.  sounds good.